### PR TITLE
Navigation menu

### PR DESCRIPTION
**Pull request by Yury-Odinson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +28,8 @@
     "gatsby-transformer-sharp": "^5.9.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-helmet": "^6.1.0"
+    "react-helmet": "^6.1.0",
+    "react-transition-group": "^4.4.5"
```
Comment (jathpr) [inline package.json:37]: if you don't use it - better to remove

File: src/components/LinkItem.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+type LinkProps = {
+    linkPage: string,
+    children: any // which type for <StaticImage />?
```
Comment (jathpr) [inline src/components/LinkItem.tsx]: children type is ReactNode

File: src/components/menu.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.menu-body {
```
Comment (jathpr) [inline src/components/menu.css]: you don't need `css`
it must be in styled components

File: src/components/Menu.tsx [hunk 4]
```diff
@@ -11,41 +15,68 @@ export const Menu = () => {
...
+    >
+        <MenuBody className="menu-body" onClick={() => close}>
+          <MenuList className="menu-list">
+            <MenuItem className="menu__item" to="/works">Works</MenuItem>
+            <MenuItem className="menu__item" to="/about">About</MenuItem>
+            <MenuItem className="menu__item" to="/contacts">Contacts</MenuItem>
+          </MenuList>
+          <Links className="menu-links">
+            <LinkItem linkPage="./">
+              <StaticImage src="../images/menu/instagram.png" alt="instagram" />
```
Comment (jathpr) [inline src/components/Menu.tsx]: for icons prefer to use svg

**Comment by Y-odinson** [comment]
I'll fix it in a few days

**Comment by Yury-Odinson** [comment]
I'll fix it in a few days

**Comment by Yury-Odinson** [comment]
done. but conflicts need to be resolved.
some conflicts are based on a version differences in branches. I don't know what version of libraries to leave - in master branch or navigation_menu branch.

**Review by jathpr — APPROVED** [review]

**Pull request merged**